src/app/shared/services/account/account.service.spec.ts

```diff
@@ -182,6 +215,39 @@ describe('AccountService', () => {
     await instance.deductAccountStorage(200);
     expect(instance.getAccount().spaceLeft).toEqual(99800);
   });
+
+  it('should send the account data to mixpanel after signing up', async () => {
+    const { instance, inject } = shallow.createService();
+    const apiService = inject(ApiService);
+    const trackerSpy = spyOn(instance, 'trackAuthWithMixpanel');
+    const account = await instance.signUp(
+      'test@permanent.org',
+      'Test User',
+      'password123',
+      'password123',
+      true,
+      true,
+      '',
+      '',
+      true
+    );
+
+    expect(trackerSpy).toHaveBeenCalled();
+  });
+
+  it('should send the account data to mixpanel after logging in', async () => {
+    const { instance, inject } = shallow.createService();
+    const apiService = inject(ApiService);
+    const trackerSpy = spyOn(instance, 'trackAuthWithMixpanel');
+    const account = await instance.logIn(
+      'test@permanent.org',
+      'password123',
+      true,
+      true
+    );
+
+    expect(trackerSpy).toHaveBeenCalled();
+  });
```

Comment:
These are some particularly fragile tests since they are relying on calling a specific function in the AccountService that are highly coupled to Mixpanel. Since they've already been written, I'm fine with keeping them, but for the future we don't really need to fully cover the code that calls out to Mixpanel in particular unless it's doing something particularly complicated.